lib/ansible/modules/net_tools/haproxy.py

```diff
@@ -251,12 +251,15 @@ def execute(self, cmd, timeout=200, capture_output=True):
         """
         self.client = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
         self.client.connect(self.socket)
-        self.client.sendall('%s\n' % cmd)
+        try:
+            self.client.sendall('%s\n' % cmd)
+        except TypeError:
+            self.client.sendall((bytearray('%s\n', 'ASCII')) % cmd.encode())
```

Comment:
We should be using `ansible.module_utils._text.to_bytes` instead of this method.

Reply:
thanks @sivel
replaced it to use `modue_utils._text`